harvester/core/models/resources/basic.py

```diff
@@ -22,8 +23,8 @@ def get_signed_absolute_uri(self, duration=7200):
         Generate a presigned URL to share the S3 object where this resource is stored.
         If the application is not connected to S3 it simply returns a local path.
         """
-        if settings.AWS_STORAGE_BUCKET_NAME is None:
-            return os.path.join(settings.MEDIA_URL, quote_plus(self.body, safe="/"))
+        if not settings.IS_AWS:
+            return "http://harvester:8080" + os.path.join(settings.MEDIA_URL, quote_plus(self.body, safe="/"))
```

Comment:
kan dit niet beter localhost zijn? anders werkt het niet als je het buiten de container om draait

Reply:
Maar localhost werkt niet vanuit de container. Deze links gaan nu naar de Tika container en vanuit die context kan het dus geen localhost zijn. Voor localhost kan je "harvester 127.0.0.1" aan je /etc/hosts toevoegen, zal dat nog even opnemen in de documentatie. Maar dat is eigenlijk alleen voor bekijken in de browser. Normaal gesproken worden deze links alleen gebruikt in de Tika container.